flickit-assessment-data/src/main/java/org/flickit/assessment/data/jpa/kit/expertgroup/ExpertGroupJpaRepository.java

```diff
@@ -1,13 +1,44 @@
 package org.flickit.assessment.data.jpa.kit.expertgroup;
 
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.query.Param;
 
+import java.util.List;
 import java.util.UUID;
 
 public interface ExpertGroupJpaRepository extends JpaRepository<ExpertGroupJpaEntity, Long> {
 
     @Query("SELECT e.ownerId FROM ExpertGroupJpaEntity as e where e.id = :id")
     UUID loadOwnerIdById(@Param("id") Long id);
+    @Query("""
```

Comment:
Add an empty line between previous method and this method

Reply:
Done